README.md

```diff
@@ -1,53 +1,171 @@
-# docker-collector-metrics
+# Docker Metrics Collector
 
-docker-collector-metrics is a Docker container that uses Metricbeat to collect metrics and forward those metrics to your Logz.io account.
+To simplify shipping metrics from one or many sources,
+we created Docker Metrics Collector.
+Docker Metrics Collector is a container
+that runs Metricbeat with the modules you enable at runtime.
 
-To use this container, you'll set environment variables in your docker run command.
-docker-collector-metrics uses those environment variables to specify which Metricbeat modules you want to be collected.
-docker-collector-metrics mounts docker.sock to the container itself, allowing Metricbeat to collect the metrics and metadata.
+Docker Metrics Collector ships metrics only.
+If you want to ship logs to Logz.io,
+see [docker-collector-logs](https://github.com/logzio/docker-collector-logs).
 
-docker-collector-metrics ships metrics only. If you want to ship logs to Logz.io, see [docker-collector-logs](https://github.com/logzio/docker-collector-logs).
-
-## docker-collector-metrics setup
+## Configuration
 
 ### 1. Pull the Docker image
 
-Download the logzio/docker-collector-metrics image:
+Download the Docker Metrics Collector image:
 
 ```shell
 docker pull logzio/docker-collector-metrics
 ```
 
 ### 2. Run the container
 
-For a complete list of options, see the parameters below the code block.👇
+For a complete list of options, see the parameters below the code block
+and in the [_Modules_](#modules) section at the bottom of this doc. 👇
 
 ```shell
 docker run --name docker-collector-metrics \
---env LOGZIO_TOKEN="<ACCOUNT-TOKEN>" \
---env LOGZIO_URL="<LISTENER-URL>" \
---env LOGZIO_MODULES="<COLLECTED-MODULES>" \
+--env LOGZIO_TOKEN="<<SHIPPING-TOKEN>>" \
+--env LOGZIO_URL="<<LISTENER-HOST>>" \
```

Comment:
Are we looking for the host only here, or the entire URL + port?

Reply:
only the url

Reply:
Actually, let's make it REGION instead and the user would insert eu, ca ...
We'll mark the LOGZIO_URL as deprecated for the sake of those who already using it this way

Reply:
Done